### PR TITLE
Remove the gipsy interpreter

### PR DESCRIPTION
**Pull request by cmickeyb**
Removes all files, references and dependencies on Gipsy. 

While Scheme still seems like a fantastic language for building smart contracts (see DAML for another commercial example), it is far easier to write in Wasm in Wawaka. This just removes a lot of the old and mostly unmaintained code.

### CODE REVIEW

File: docker/Dockerfile.pdo-dev [hunk 1]
```diff
@@ -87,7 +87,6 @@ RUN apt-get update \
     libsecp256k1-dev \
     libssl-dev \
     libtool \
-    tinyscheme \
```
Comment (bvavala) [inline docker/Dockerfile.pdo-dev:90]: I'm wondering if there's any dependency here due to tinyscheme -- difficult to figure it out without trying to remove any of them.

**Review by bvavala — APPROVED** [review]
Looks good, tested.

File: docker/Dockerfile.pdo-dev (hunk 1 above)
Comment (cmickeyb) [reply docker/Dockerfile.pdo-dev:90]: Not entirely sure I understand the comment. But tinyscheme has no other dependencies.

**Review by prakashngit — APPROVED** [review]
thank you, looks good

File: client/etc/sample_client.toml [hunk 2]
```diff
@@ -78,11 +78,10 @@ Duration=120 #seconds
...
 
-# Name is the name of the contract class in the
-# Gipsy Scheme contract
+# Name is the name of the contract class
 Name = "${contract}"
 
-# SourceFile is the name of the Gipsy Scheme
+# SourceFile is the name of the
 # source file that defines the contract class
 SourceFile = "${contract}.scm"
```
[the remark this answers is not in the document]
Comment (cmickeyb) [reply client/etc/sample_client.toml:86]: done

**Review by marcelamelara — APPROVED** [review]
Thanks! LGTM.

**Pull request merged**